src/cloud-api-adaptor/test/e2e/common.go

```diff
@@ -26,6 +27,14 @@ const BUSYBOX_IMAGE = "quay.io/prometheus/busybox:latest"
 const WAIT_DEPLOYMENT_AVAILABLE_TIMEOUT = time.Second * 180
 const DEFAULT_AUTH_SECRET = "auth-json-secret-default"
 
+func isTestWithKbs() bool {
+	return os.Getenv("DEPLOY_KBS") == "true" || os.Getenv("DEPLOY_KBS") == "yes"
+}
+
+func isTestWithCustomizedOpa() bool {
+	return os.Getenv("TEE_CUSTOMIZED_OPA") != ""
```

Comment:
Is is worth checking if this file exists and erroring if not? I'm guessing the failures I saw yesterday might have been because I set `TEE_CUSTOMIZED_OPA` relative, not absolute, so it didn't find it?

Reply:
Sorry, I just spotted `os.Stat(customizedOpaFile)` elsewhere, which I think should handle this

Reply:
Fixed, thanks!